xml/System.Diagnostics/Activity.xml

```diff
@@ -1204,11 +1209,20 @@ This method has the advantage that no string manipulation is needed to set the I
         <Parameter Name="description" Type="System.String" Index="1" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="code">To be added.</param>
-        <param name="description">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="code">The status code</param>
+        <param name="description">The error status descrition</param>
+        <summary>Sets the status code and description on the current activity object.</summary>
+        <returns>'this' for convenient chaining</returns>
```

Comment:
><returns>'this' for convenient chaining</returns> [](http://example.com/codeflow?start=8&length=49)

<returns><see langword="this" /> for convenient chaining.</returns>

Reply:
this change is just the results of the automated port tool, I think you are proposing this should be changed in the source?